ddtrace/opentracer/tracer.py

```diff
@@ -1,13 +1,69 @@
 import opentracing
+import logging
+
+from ddtrace import Tracer as DatadogTracer
+from ddtrace.constants import FILTERS_KEY
+from ddtrace.ext import AppTypes
+from ddtrace.settings import ConfigException
+
+from .settings import ConfigKeys as keys, config_invalid_keys
+
+from .util import merge_dicts
+
+
+log = logging.getLogger(__name__)
+
+DEFAULT_CONFIG = {
+    keys.AGENT_HOSTNAME: 'localhost',
+    keys.AGENT_PORT: 8126,
+    keys.DEBUG: False,
+    keys.ENABLED: True,
+    keys.GLOBAL_TAGS: {},
+    keys.SERVICE_NAME: None,
+    keys.SAMPLER: None,
+    keys.APP_TYPE: AppTypes.worker,
+    keys.CONTEXT_PROVIDER: None,
+    keys.PRIORITY_SAMPLING: None,
+    keys.SETTINGS: {
+        FILTERS_KEY: [],
+    },
+}
 
 
 class Tracer(opentracing.Tracer):
-    """"""
+    """A wrapper providing an OpenTracing API for the Datadog tracer."""
 
-    __slots__ = []
+    __slots__ = ['_enabled', '_debug', '_service_name', '_tracer']
 
-    def __init__(self, scope_manager=None):
-        pass
+    def __init__(self, config={}, service_name=None, scope_manager=None):
```

Comment:
* the `scope_manager` can be ignored at the moment
* we should keep track of the fact that `service_name` must have a default that is not `None`. Fine to avoid doing something now, but let's keep track somewhere

Reply:
I'll throw a comment in the code to make sure we set a default later on.